### PR TITLE
PLANET-6901 Move Page Header metabox fields to sidebar

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-6901](https://jira.greenpeace.org/browse/PLANET-6901)
These settings used to be in the metabox (see [related PR](https://github.com/greenpeace/planet4-master-theme/pull/1803) to remove them from there) but we move them to the sidebar in order to keep all settings in the same place

### Testing

You can test the changes in any page or campaign, either on local or on the tavros test instance!

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [tavros](https://www-dev.greenpeace.org/test-tavros) | [admin](https://www-dev.greenpeace.org/test-tavros/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-tavros/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-tavros) | [composer-local.json](https://github.com/greenpeace/planet4-test-tavros/blob/main/composer-local.json)

:watch: 2022.09.26 15:32:12

**Comment by sagarsdeshmukh** [comment]
Hi @mleray , The background image for the page header fields looks like not getting picked up, could you please check at [here](https://www-dev.greenpeace.org/test-tavros/test-page/), the same data added on other instance and the bg image looks like [this](https://www-dev.greenpeace.org/test-titan/test-page/).

File: assets/src/setupCustomSidebar.js [hunk 1]
```diff
@@ -1,45 +1,48 @@
...
-        render: ActionSidebar,
-      };
-    default:
-      return null;
+const sidebarsForPostType = postType => {
+  switch (postType) {
+  case 'campaign':
+    return [
+      PageHeaderSidebar,
+      CampaignThemeSidebar,
```
Comment (sagarsdeshmukh) [inline assets/src/setupCustomSidebar.js:11]: Could you please confirm, both panels are visible on campaign page, or might be my local issue 🤔

**Review by sagarsdeshmukh — APPROVED** [review]
Looks good to me 👍

**Pull request merged**